spiped/spiped.1

```diff
@@ -153,5 +153,13 @@ Print version number.
 .TP
 .B \-1
 Exit after a connection closes.  Used for internal testing.
+.SH SIGNALS
+.B SIGTERM
```

Comment:
@gperciva Oh, nevermind. I found the formatting mistake, force pushing any second.